packages/utils/logger/src/index.ts

```diff
@@ -1,9 +1,7 @@
-'use strict';
+import winston from 'winston';
 
-const winston = require('winston');
-
-const formats = require('./formats');
-const configs = require('./configs');
+import * as formats from './formats';
+import * as configs from './configs';
 
 const createLogger = (userConfiguration = {}) => {
```

Comment:
At some point, we should open the debate in the TS guidelines about whether we want to prefer inferring the return types for this kind of util or declaring it manually to enforce it.